src/consensus/pbft/libbyz/nodeinfo.h

```diff
@@ -41,6 +41,7 @@ struct GeneralInfo
   long view_timeout;
   long status_timeout;
   long recovery_timeout;
+  uint64_t max_requests_between_signatures;
```

Comment:
should this be called `sig_max_tx` like it is in the Raft configuration?

Reply:
These settings should be reused and shared once the front end triggers the signatures for both.

Reply:
#608 - I need to think about this. I don't have a good solution right now :S